# Desired distance from grasp_postures_frame_id to put the object
# So, the grasping point
grasp_desired_distance: 0.10
grasp_min_distance: 0.0
# Times for the grasp, they are sumed step after step
time_pre_grasp_posture: 1.2
time_grasp_posture: 0.5
time_grasp_posture_final: 0.5
# Base frame of MoveIt for your robot
grasp_pose_frame_id: base_link
# Frame where the direction of the poses will be computed
grasp_postures_frame_id: crane_x7_gripper_base_link
# Currently unused grasp quality to put in the message
grasp_quality: 0.1
# Gripper joint poses
gripper_joint_names: "crane_x7_gripper_finger_a_joint crane_x7_gripper_finger_b_joint"
# Closed position
gripper_grasp_positions: "0.18 0.18"
# Open position
gripper_pre_grasp_positions: "0.70 0.70"

max_contact_force: 0.0
allowed_touch_objects: ''

# Rotation to apply if you have unaligned
# tool link to grasping frame
fix_tool_frame_to_grasping_frame_roll: -90.0
fix_tool_frame_to_grasping_frame_pitch: 0.0
fix_tool_frame_to_grasping_frame_yaw: 0.0

# Generation of grasps configuration
# in order to generate more or less poses these
# parameters can be tuned
min_degrees_yaw: 0
max_degrees_yaw: 360
min_degrees_pitch: 0
max_degrees_pitch: 360

# The steps are the main thing to tune
step_degrees_pitch: 15
step_degrees_yaw: 15

links_to_allow_contact: ["crane_x7_gripper_finger_a_joint", "crane_x7_gripper_finger_b_joint", "crane_x7_gripper_base_link"]
